src/mixins/Json2Vue.js

```diff
@@ -2,7 +2,7 @@ import extensions from './extensions';
 import ScreenBase from './ScreenBase';
 import CountElements from '../CountElements';
 import ValidationsFactory from '../ValidationsFactory';
-import _, { debounce, isEqual } from 'lodash';
+import _, { isEqual } from 'lodash';
```

Comment:
if you can, do a find a replace of all the `_.` and add the imports of the properties we are using from lodash

Reply:
not a blocker